GPTsettings.py

```diff
@@ -49,10 +48,9 @@ def a_func(a:int, b:int):
     Some desired properties:
         - When specifying the positions, Use the right line for the comment as specified in the diff
         - Be really careful with this! Don't use the ' token in your comments (only when needed for the code) or it will break the python code.
-        - Provide code suggestions as much as possible in your comments, for that you can use the standart format ```suggestion <suggested_change> ```
+        - Provide code suggestions as much as possible in your comments, for that you can use the standart format ```suggestion \n <suggested_change> \n```
```

Comment:
Why break the space-time continuum with new lines in markdown code blocks? Those backticks with `suggestion` need to cuddle their content more closely, trust me, it'll look better. Keep it tight, nobody likes extra whitespace: ```suggestion
<your_code_here>
```.